lib/Metrc/client.rb

```diff
@@ -101,6 +101,16 @@ def list_strains(license_number)
       list(:strains, license_number)
     end
 
+    def list_transfer_templates(license_number, range_start: nil, range_end: nil)
+      query_string = [].tap do |arr|
+        arr << "licenseNumber=#{license_number}"
+        arr << "lastModifiedStart=#{range_start}" unless range_start.nil?
+        arr << "lastModifiedEnd=#{range_end}" unless range_end.nil?
+      end.join('&')
```

Comment:
What if we use the URI#encode_www_form module?
```suggestion
      query_string = URI.encode_www_form([['licenseNumber', license_number], ['lastModifiedStart', range_start], ['lastModifiedEnd', range_end]])
```

Reply:
Good catch - I've updated that. Slightly different from your suggestion as we need to exclude date params entirely if they're not present.